### PR TITLE
Remove Home page and replace with Extract New page

### PR DESCRIPTION
**Pull request by jafeltra**
This PR removes the `Home` page and replaces it with the `Extract New` page.

All the paths that directed to `/extract` have been replaced with `/` since that is the new home page. If we think that in the future we might want a different landing/home page or we like the descriptiveness of `/extract`, I can revert those changes and just redirect `/` to `/extract` for now. This was another approach I considered and didn't end up going with, but is very possible.

### CODE REVIEW

File: react-app/src/components/Extract.js [hunk 1]
```diff
@@ -153,7 +152,7 @@ function Extract(props) {
             </Row>
           </Form>
           <div className="nav-button-container">
-            <LinkButton className="generic-button" size="lg" variant="outline-secondary" text="Cancel" path="/" />
+            <div></div>
```
Comment (Dtphelan1) [inline react-app/src/components/Extract.js:155]: DP1 - Do we need this `div`?

**Comment by Dtphelan1** [comment]
DP2 - Can we change the text on the "Back" button below to something more indicative? Do we even want a back button here anymore? I'd argue we might not, right – since the Config Editor is on it's own page now?

image

**Comment by jafeltra** [comment]
DP1 and DP2 are addressed in 78a1c2f

**Comment by Dtphelan1** [comment]
> What is but one more pixel?  
Everything

**Review by Dtphelan1 — APPROVED** [review]

**Pull request merged**